apps/demo-cra-lib/package.json

```diff
@@ -16,7 +16,9 @@
   "scripts": {
     "start": "react-app-rewired start",
     "build": "react-app-rewired build",
-    "lint": "eslint \"**/*.{js,cjs,ts,tsx}\" --max-warnings=0"
+    "lint": "run-p lint:eslint lint:tsc",
```

Comment:
I wouldn't bother with these `lint` scripts in all the projects (especially since it requires installing `npm-run-all`. The important bit is that we have one command at the root that runs everything, so I'd instead rename `lint:all` to `lint` in the root `package.json`. (It's great that we're freeing up the bare name `lint` in the root like this!)